Solutions/Cybersixgill-Actionable-Alerts/Playbooks/CybersixgillAlertStatusUpdate/azuredeploy.json

```diff
@@ -0,0 +1,513 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Cybersixgill-Alert-Status-Update",
+        "description": "This playbook will update status of Cybersixgill Alerts when respective incident status is updated in Microsoft Sentinel",
+        "prerequisites": "",

```

Comment:
Enter something in prerequisites that client id and secret key should be stored in key vault before deploying this playbook .